.env

```diff
@@ -0,0 +1 @@
+HTTPPORT=80
```

Comment:
New commit fixes this so there's consistency and most people won't have to fiddle with environment vars unless they're doing a demo (where "http://localhost" obviously won't work)